samples/algorithms/GHZ.qs

```diff
@@ -30,23 +31,18 @@ namespace Sample {
         DumpMachine();
 
         // Measure and reset qubits before releasing them.
-        let results = MeasureEachZ(qs);
-        ResetAll(qs);
-        return results;
+        MResetEachZ(qs)
     }
 
     /// # Summary
-    /// This operation prepares a generalized GHZ state across a register of qubits.
-    ///
-    /// # Input
-    /// ## qs
-    /// The given register of qubits to be transformed into the GHZ state. It is assumed
-    /// that these qubits are in their default |0〉 state.
+    /// Prepares state (|000〉 + |111〉) / √2 (GHZ state) across a register
+    /// of three qubits `qs`.
+    /// All qubits are assumed to be in |0〉 state on input.
     operation PrepareGHZState(qs : Qubit[]) : Unit {
-        Fact(Length(qs) > 0, "`qs` length must be greater than zero");
-        Fact(CheckAllZero(qs), "All qubits must be in the |0〉 state");
+        Fact(Length(qs) == 3, "`qs` length be 3.");
```

Comment:
```suggestion
Fact(Length(qs) == 3, "`qs` length should be 3.");
```

Reply:
A word dropped out on the way from my brain to keyboard...